python/src/otel/otel_sdk/requirements.txt

```diff
@@ -1,3 +1,3 @@
-opentelemetry-exporter-otlp-proto-http==1.19.0
-opentelemetry-distro==0.40b0
-opentelemetry-instrumentation-aws-lambda==0.40b0
+opentelemetry-exporter-otlp==1.20.0
```

Comment:
Note to self, this seems to be needed by APM Python layer to use `otlp` default entry point for `EXPORTER` config.